eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FullText.php

```diff
@@ -178,18 +201,26 @@ protected function getWordIdSubquery( SelectQuery $query, $string )
             $wordExpressions[] = $this->getWordExpression( $subQuery, $token );
         }
 
+        // Create where expression
+        $whereCondition = $subQuery->expr->lOr( $wordExpressions );
+
+        // If stop word threshold is below 100% we make sure it is applied
+        // @see getStopWordThresholdValue()
+        if ( $this->configuration['stopWordThresholdFactor'] < 1 )
+        {
+            $whereCondition = $subQuery->expr->lAnd(
```

Comment:
hmmm, is this right that `$whereCondition` is overwritten right after it has been defined ?